test/protocol-tests-core/src/main/java/software/amazon/awssdk/protocol/asserts/marshalling/XmlAsserts.java

```diff
@@ -71,11 +73,18 @@ private static String formatXml(String xmlDocumentString) throws Exception {
     }
 
     private static String formatXml(Document xmlDocument) throws Exception {
-        Transformer transformer = TransformerFactory.newInstance().newTransformer();
+        Transformer transformer = transformerFactory().newTransformer();
         transformer.setOutputProperty(OutputKeys.INDENT, "yes");
         StreamResult result = new StreamResult(new StringWriter());
         DOMSource source = new DOMSource(xmlDocument);
         transformer.transform(source, result);
         return result.getWriter().toString();
     }
+
+    private static TransformerFactory transformerFactory() throws TransformerConfigurationException {
```

Comment:
XmlUtils?

Reply:
This is only used in unit tests to format the xml and not expected to be used anywhere else. I'll move the other one to `XmlUtils`